### PR TITLE
COMMERCE-4088 Fix language key

### PR DESCRIPTION
**Pull request by alecsloan**
Suppression is needed for an SF isssue :  I mentioned to Hugo: https://liferay.slack.com/archives/C1SJ64S9X/p1600206039024800

### CODE REVIEW

**Comment by alecsloan** [comment]
ci:test:sf

**Comment by alecsloan** [comment]
ci:test:relevant

File: modules/dxp/apps/commerce/commerce-organization-web/source-formatter-suppressions.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0"?>
+
+<suppressions>
+	<source-check>
+		<suppress checks="LanguageKeysCheck" files="src/main/java/com/liferay/commerce/organization/web/internal/frontend/CommerceOrganizationClayTableDataSetDisplayView\.java" />
```
Comment (dsitu) [inline modules/dxp/apps/commerce/commerce-organization-web/source-formatter-suppressions.xml:5]: @alecsloan im not sure about adding this suppression, is there reason why we dont just add that language key?

Comment (alecsloan) [reply modules/dxp/apps/commerce/commerce-organization-web/source-formatter-suppressions.xml:5]: we already have the language key in CE and it's usable, but SF doesn't seem to detect it. I mentioned to Hugo: https://liferay.slack.com/archives/C1SJ64S9X/p1600206039024800

Comment (dsitu) [reply modules/dxp/apps/commerce/commerce-organization-web/source-formatter-suppressions.xml:5]: ahhh ic kk, sounds good to me. thanks!

**Review by dsitu — APPROVED** [review]

**Comment by marco-leo** [comment]
ci:forwad

**Comment by marco-leo** [comment]
ci:forward

**Pull request closed without merging**